MathMagician/Program.cs

```diff
@@ -3,13 +3,75 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using MathMagician.Numbers;
 
 namespace MathMagician
 {
     class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("What would you like for me to do?");
+            Console.WriteLine("Press 1 to print integers");
+            Console.WriteLine("");// "How" you provide the user useful info is on you.
+            // Think about, "How will the user pick a command?"
+            // Is it better to modify the original question?  Make it more useful?
+
+            string command = Console.ReadLine();
+            int formattedCommand = Int32.Parse(command);
+            // Once I have the command (whatever it looks like), how do I check if it's a valid command?
```

Comment:
Take a look at [TryParse](https://msdn.microsoft.com/en-us/library/f02979c7(v=vs.110).aspx). It gives you a way to check of the conversion from `string` to `int` succeeded.